tests/testBurrow.ml

```diff
@@ -644,6 +752,46 @@ let suite =
             (Ligo.mul_int_int lp_den f_den))
     );
 
+    (* TODO: Property "compute_tez_to_auction - <= 0 for underburrowed burrows" *)
+    (* TODO: Property "compute_tez_to_auction - > 0 for overburrowed burrows" *)
+
+    ("compute_tez_to_auction - expected value for an overburrowed burrow" >::
+     fun _ ->
+       let burrow = Burrow.make_burrow_for_test
+           ~outstanding_kit:(kit_of_mukit (Ligo.nat_from_literal "3n"))
+           ~excess_kit:kit_zero
+           ~active:true
+           ~address:alice_addr
+           ~delegate:None
+           ~collateral:(Ligo.tez_from_literal "3mutez")
+           ~adjustment_index:fixedpoint_one
+           ~collateral_at_auction:(Ligo.tez_from_literal "3mutez")
+           ~last_touched:(Ligo.timestamp_from_seconds_literal 0) in
+       (* Note: cranking up the index to make test more sensitive to small changes potentially obscured by rounding *)
+       let parameters = Parameters.({initial_parameters with index=(Ligo.tez_from_literal "100_000_000_000mutez");}) in
```

Comment:
Hmmm, setting the index directly like this (instead of by touching the parameters for example) makes the difference between the index and the protected index somewhat "impossible" under normal circumstances:
```ocaml
index           = Ligo.tez_from_literal "100_000_000_000mutez"
protected_index = Ligo.tez_from_literal       "1_000_000mutez"
```
I guess it's OK since we're writing this test only to address the mutation, but in general setting fields in an ad-hoc way like this can lead to cases that are otherwise impossible to trigger.

Perhaps add a comment here about the distance between the index and the protected index for our future sanity :sweat_smile:

Reply:
Thinking more about this, I guess we can get the index to be very far from the protected index, if the oracle is compromised. Perhaps setting it explicitly is not that bad in this sense after all.